x/feegrant/keeper/keeper.go

```diff
@@ -244,23 +244,19 @@ func (k Keeper) UseGrantedFees(ctx context.Context, granter, grantee sdk.AccAddr
 	}
 
 	remove, err := grant.Accept(ctx, fee, msgs)
-
 	if remove {
 		// Ignoring the `revokeFeeAllowance` error, because the user has enough grants to perform this transaction.
-		k.revokeAllowance(ctx, granter, grantee)
+		_ = k.revokeAllowance(ctx, granter, grantee)
```

Comment:
shouldn't this line be like  
```suggestion
		err = k.revokeAllowance(ctx, granter, grantee)
```

Reply:
Given the comment above I do not think so.

Reply:
hmm, but we are checking for the no err in L250? that is not needed any more if we are not catching the error here.

Reply:
It is needed as error L246 still needs to be checked. However, this can be improve: see https://github.com/cosmos/cosmos-sdk/pull/16406#discussion_r1251963764.